src/SfxWeb/src/app/views/cluster/essentials/essentials.component.ts

```diff
@@ -91,6 +99,12 @@ export class EssentialsComponent extends BaseControllerDirective {
       this.nodes.refresh(messageHandler).pipe(map(() => {this.updateItemInEssentials(); })),
       this.systemApp.refresh(messageHandler).pipe(catchError(err => of(null))),
       this.clusterUpgradeProgress.refresh(messageHandler),
+      this.RestClient.getPartitionById(SystemServicePartitionId.FailoverManagerId, messageHandler).pipe(map((partition) => {
+        if (partition.PartitionStatus === 'InQuorumLoss') {
```

Comment:
this should be like
this.fmQuorumLossStatus = partition.PartitionStatus === 'InQuorumLoss'

because as it stands now once its set once it will never clear the warning even if it goes out of quorum loss